<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Основное окно</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ShowDialogATV12">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>частотники</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowDialogCE303">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CE303</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowDialogI2C">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>I2C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Power">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Alarm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pavilion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pavilion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Nasos1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pict/pict/nasosgray.gif);</string>
    </property>
    <property name="text">
     <string>N1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Nasos2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pict/pict/nasosgray.gif);</string>
    </property>
    <property name="text">
     <string>N2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Nasos3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pict/pict/nasosgray.gif);
color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>N3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Nasos4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pict/pict/nasosgray.gif);</string>
    </property>
    <property name="text">
     <string>N4</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pict/pict/popl_4_uroven_avariya.gif);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelLevel">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>110</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>55</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);background-image: url(none);</string>
     </property>
     <property name="text">
      <string>45</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="setup">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Установки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="service">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Сервис</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
